backend/src/storage/storage.module.ts

```diff
@@ -0,0 +1,39 @@
+import { S3Client, S3ClientConfig } from "@aws-sdk/client-s3";
+import { Module } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+
+const s3ClientFactory = {
+	provide: "STORAGE_CLIENT",
+	useFactory: (configService: ConfigService): S3Client | null => {
+		if (configService.get<boolean>("FILE_UPLOAD") === false) {
+			return null;
+		}
+		const region = configService.get<string>("AWS_REGION") || "us-east-1";
```

Comment:
Why the default region is `us-east-1`?

Reply:
chose `us-east-1` as the default region because it's AWS's first and one of the most commonly used regions. The goal is to provide a stable, well-known fallback for cases where the user doesn't specify a region.

Reply:
I believe it should be empty so the S3 client can throw an error. What do you think?

Reply:
I also agree that users may not be able to check if they are set up with us-east-1.